source/adapters/level_zero/device.cpp

```diff
@@ -900,6 +910,17 @@ ur_device_handle_t_::useImmediateCommandLists() {
   }
 }
 
+bool ur_device_handle_t_::useLargeAllocations() {
+  static const bool UseLargeAllocations = [this] {
+    const char *UrRet = std::getenv("UR_L0_ALLOW_LARGE_ALLOCATIONS");
+    if (!UrRet)
+      return (this->isPVC() ? true : false);
```

Comment:
why this isPVC check ? 
It is not required on PVC.
PVC Level Zero driver does it by default.

Reply:
@MichalMrozek : this is so the rest of the adapter has  "large allocation behavior".

Reply:
But PVC is already in large allocation behavior by default.
There is no need to do any special for this device.
There is no need to add compiler option and it is quite dangerous to bypass max memory allocation size limits by default.
By using those variables it indicates that PVC requires some special handling for large allocations which in fact is not needed.

Reply:
thanks @MichalMrozek . I have changed code to use defaults on PVC.